src/app/components/TaqvimTable/index.js

```diff
@@ -14,15 +14,12 @@ export default function TaqvimTable() {
     return localStorageData ? JSON.parse(localStorageData) : {};
```

Comment:
@riyashivolkar this line of code is not working during server side rendering because LocalStorage api is not available untill html is served by server. so that preview build failed because of that.

instead do like this

`const [isChecked, setIsChecked] = useState({})`

`useEffec(() => {
    setIsChecked(JSON.parse(localStorage?.getItem("checkboxData)));
}, [])`

so after SSR, useEffect helps to set initial value of isChecked in the initial render.

Reply:
and also, some people started using this app so please avoid getting undefined thrown. this can interrupt users using our app. Please consider this during development